CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+CONTRIBUTING
+============
+
+Contributions are always welcome, no matter how large or small. Before contributing,
+please read the [code of conduct](code-of-conduct.md).
+
+## Setup
+
+#### Prerequisites
+
+You need to have [Node.js](https://nodejs.org/en/), [NVM](https://github.com/creationix/nvm), [NPM](https://www.npmjs.com/) and [Jekyll](https://jekyllrb.com/) installed on your computer.
+
+```
+$ node --version
+v4.6.0
+
+$ npm --version
+2.15.9
+
+$ jekyll --version
+jekyll 3.1.2
+```
+
+
+#### Fork our repo
+
+1. Fork the repo: https://github.com/opentracing/opentracing.io
+1. Run the following commands:
+
+```
+$ git clone https://github.com/<YOUR_USERNAME>/opentracing.io
+$ cd opentracing.io
+$ nvm use
+$ npm install
+```
+
+## Running Opentracing.io locally
+
+```
+$ npm run dev
+$ open http://localhost:4000
+```
+
+<details>
```

Comment:
There is currently no way of creating <details>-like functionality with markdown.
I wanted to hide this info by default since it's a lot of text, and I think the majority of users won't care about it. 

But can remove it, I have no strong opinions here. 🤔